content/components/side-nav/rationale/three-levels.md

```diff
@@ -3,6 +3,13 @@ layout: component/rationale.js
 heading: Three levels
 ---
 
-Each level in the side nav needs to be easily visually identifiable as a new level, beyond 3 levels visual solutions quickly become too complex or drastically reduce the available line-length for links.
+Each level in the side nav needs to be easily visually identifiable as a new level. Beyond 3 levels, visual solutions quickly become too complex or drastically reduce the available line length for links.
 
-After reviewing multiple government and private sector websites it was found that the upper extent of complex navigation solutions tended to be around 5 levels. To avoid overly complex user interfaces the navigation is often split into various groups. Typically 2 levels in the main-navigation, and 3 in side navigation, supported by listing/landing pages if more levels are required.
+After reviewing multiple government and private sector websites, it was found that the upper extent of complex navigation solutions tended to be around 5 levels. To avoid overly complex user interfaces the navigation is often split into groups.
+
+Typically:
+
+- 2 levels in the main navigation, and
```

Comment:
I think this is incorrect, one level in main navigation with a landing page containing level two items